### PR TITLE
treewide: re-uncrustify whitelisted files

### PR DESCRIPTION
**Pull request by miri64**
### Contribution description
Newer versions of `uncrustify` provide more patch suggestions then the once applied in https://github.com/RIOT-OS/RIOT/pull/10867 and https://github.com/RIOT-OS/RIOT/pull/13975. I also applied the discussed fixes from https://github.com/RIOT-OS/RIOT/pull/15778#issuecomment-760936289 (I hope I understood @bergzand correctly :sweat_smile:)

### Testing procedure
`uncrustify` should pass in the `static-tests` action. The binaries should not change.

### Issues/PRs references
None

### CODE REVIEW

**Comment by miri64** [comment]
I just noticed, that there was an error in the `whitelist.txt` with regards to `sys/include/ztimer.h` (which was described as `sys/ztimer.h`. I added a fix for that and also uncrustified `sys/include/ztimer.h`

**Comment by fjmolinas** [comment]
Static-check still shows two comments.

**Comment by miri64** [comment]
> Static-check still shows two comments.

Yeah, but those are false positives, that are fixed in my (later) version of `cppcheck`.

File: core/msg.c [hunk 1]
```diff
@@ -242,7 +243,7 @@ int msg_send_bus(msg_t *m, msg_bus_t *bus)
     int count = 0;
 
     m->sender_pid = (in_irq ? KERNEL_PID_ISR : thread_getpid())
-                  | MSB_BUS_PID_FLAG;
+                    | MSB_BUS_PID_FLAG;
```
Comment (benpicco) [inline core/msg.c:246]: Is this really what we want?

Comment (miri64) [reply core/msg.c:246]: I just used `uncrustify` and @kaspar030 said, changes made by `uncrustify` are non-negotiable.

Comment (miri64) [reply core/msg.c:246]: + I actually would prefer it as proposed by `uncrustify` (aligning RHS of `=` operator).

**Comment by miri64** [comment]
I also took the liberty and fixed some stuff pointed out by `vera++`.

**Comment by miri64** [comment]
Rebased due to https://github.com/RIOT-OS/RIOT/pull/15857 being merged caused a conflict in this PR. Can I have a review soonish? There shouldn't be much to discuss :-/.

**Review by kaspar030 — APPROVED** [review]
ACK. Visually verified these are only style changes.

**Comment by miri64** [comment]
Squashed

**Pull request merged**